src/java_bytecode/object_factory_parameters.h

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************\
+
+Module:
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_JAVA_BYTECODE_OBJECT_FACTORY_PARAMETERS_H
+#define CPROVER_JAVA_BYTECODE_OBJECT_FACTORY_PARAMETERS_H
+
+#include <cstdint>
+#include <limits>
+
+#define MAX_NONDET_ARRAY_LENGTH_DEFAULT 5
+#define MAX_NONDET_STRING_LENGTH std::numeric_limits<std::int32_t>::max()
```

Comment:
should be `std::numeric_limits<std::size_t>::max`

Reply:
Not so actually, Java Strings are bounded by Java int, i.e. signed int32.

Reply:
Maybe add this as a comment to cover this trap.